File: Exercises/C++/Dietel & Dietel C++ How to Program/Chapter_2/Exercises/Exercise2_16_Arithmetic/arithmetic/arithmetic/arithmetic.cpp
// arithmetic.cpp : This file contains the 'main' function. Program execution begins and ends there.
//

#include <iostream>
using namespace std;

int main()
{
    int number1;
    int number2;
    int sum;
    int product;
    int difference;
    int quotient;

    cout << "Enter the first integer: ";
    cin >> number1;

    cout << "Enter the second integer: ";
    cin >> number2;

    sum = number1 + number2;
    difference = number1 - number2;
    product = number1 * number2;
    quotient = number1 / number2;

    cout << "The sum of " << number1 << " and " << number2 << " is " << sum << endl;
    cout << "The difference of " << number1 << " and " << number2 << " is " << difference << endl;
    cout << "The product of " << number1 << " and " << number2 << " is " << product << endl;
    cout << "The quotient of " << number1 << " and " << number2 << " is " << quotient << endl;

}

File: Exercises/Java/Java_How to Program_8th Edition/Visual Studio/Chapter 2_Introduction to Java Applications/Exercises/Arithmetic/Arithmetic/src/main/java/pkg/Arithmetic.java
package pkg;

import java.util.Scanner;

public class Arithmetic 
{
	public static void main(String[] args) 
	{
		int number1;
		int number2;
		int sum;
		int product;
		int difference;
		int quotient;

		Scanner input = new Scanner(System.in);

		System.out.print("Enter the first integer: ");
		number1 = input.nextInt();

		System.out.print("Enter the second integer: ");
		number2 = input.nextInt();

		sum = number1 + number2;
		product = number1 * number2;
		difference = number1 - number2;
		quotient = number1 / number2;

		System.out.printf("Sum is %d\n", sum);
		System.out.printf("Product is %d\n", product);
		System.out.printf("Difference is %d\n", difference);
		System.out.printf("Quotient is %d\n", quotient);
	}
}
